adaptations/KawaiK5000.py

```
@@ -0,0 +1,588 @@
+    def send(self, message: List[int]) -> List[int]:
+        # Check which message is sent to us, and produce replies
+        if message[:7] == [0xF0, KawaiSysexID, self.channel, OneBlockDumpRequest, 0x00, 0x0a, 0x00]:
+            # This is a program dump request, let's see which bank and patch
+            bank_byte = message[7]
+            patch_number = message[8]
+            if patch_number in self.test_data.all_messages:
+                return self.test_data.all_messages[patch_number]
+            else:
+                return []
+        elif message[:7] == [0xF0, KawaiSysexID, self.channel, AllBlockDumpRequest, 0x00, 0x0a, 0x00]:
+            bank_byte = message[7]  # 0x00 would be A, 0x02 is D, 0x03 is E, 0x04 is F
```

Ruff (0.15.1)

[error] Local variable `bank_byte` is assigned to but never used

Remove assignment to unused variable `bank_byte`

(F841)